src/lib/geojson.js

```diff
@@ -0,0 +1,17 @@
+export const geojson = (timeseries) => ({
```

Comment:
You are mapping timeseries into a new format. So I would rename this file to `map-timeseries-to-geojson` and just export it as a default function like the other functions in the `lib` folder.